src/base/rss/rss_article.cpp

```diff
@@ -112,6 +112,11 @@ void Article::markAsRead()
     }
 }
 
+void Article::markAsUnread()
```

Comment:
I actually think this is a useful feature. It is something I had been considering adding as well, so I don't mind it being added as part of this.